recipe/conda_build_config.yaml

```diff
@@ -52,13 +52,15 @@ pin_run_as_build:
 arpack:
   - 3.5
 boost:
-  - 1.65.1
+  - 1.66.0
+boost-cpp:
+  - 1.66.0
 bzip2:
   - 1.0
 cairo:
   - 1.14
 curl:
-  - 7.54
+  - 7.44
 dbus:
   - 1
```

Comment:
This was not present in `pin_the_slow_way.py`. Given it seems pretty stable and has been at 1 a long time, I don't know if we need this, but I guess it can't hurt.